cfme/base/ui.py

```diff
@@ -326,14 +328,12 @@ def step(self):
 
 @navigator.register(Server)
 class About(CFMENavigateStep):
+    VIEW = AboutView
     prerequisite = NavigateToSibling('LoggedIn')
 
     def step(self):
-        if self.obj.appliance.version > '5.7':
-            from cfme.dashboard import click_help
-            click_help('About')
-        else:
-            self.prerequisite_view.navigation.select('Settings', 'About')
+        from cfme.dashboard import click_help
```

Comment:
Noted - There was no comment, so I'm not sure why originally it was imported here and not at the top.  Usually done to avoid circular imports AFAIK. I'll dig into it some more and see if we can move the import up, since its used in both here and in Documentation nav.